### DISCUSSION TITLE
建议加入课程批量下载功能

### DISCUSSION THREAD

**Discussion by bakeham** [the1812/Bilibili-Evolved#1348] [功能建议 / Ideas]
如链接：https://www.bilibili.com/cheese/play/ep850
![image](https://user-images.githubusercontent.com/49135636/102842802-c9da7680-4442-11eb-92a4-b15ee5369c72.png)

单个文件可以下载，但是无法批量下载，即使已经购买了此课程

**Comment by the1812** [comment]
连开3个干什么...  课程目前是不支持批量的

**Reply by bakeham** [reply]
首先感谢您的回复
抱歉，发了discussions，以为是issue，在里面找不到我以为是我网络问题没有发成功，就去提bug了；这边discussions没法关闭，麻烦您调整:-)

**Reply by the1812** [reply]
那我先关闭其他重复的了, 但这个功能(属于下载视频相关的新功能)短期内不会添加, 会留在讨论区里

**Reply by bakeham** [reply]
好的，麻烦您了

**Reply by traition** [reply]
呜呜呜好想要这个功能

**Reply by traition** [reply]
https://www.bilibili.com/cheese/play/ep6931
这里附加功能没有下载了。
或者说所有的课程页的附加功能都没有下载了
